docs/glossary.rst

```diff
@@ -0,0 +1,55 @@
+        An alternative term for a passkey or resident key. It refers to a WebAuthn credential that allows authentication without the server providing credential IDs.
+
+    key pair
+        A public-private pair of cryptographic keys used for secure communication and authentication. The public key can be shared freely, while the private key is kept secret and used to decrypt data or create a digital signature.
+
+        The two keys are mathematically linked. So data encrypted with the public key can only be decrypted by the corresponding private key, and vice versa. This ensures secure interactions without needing to share the private key.
+
+    passkey
+        A type of WebAuthn credential that supports passwordless logins in scenarios where the server has not first identified the user and has not provided the web browser with credential IDs.
+
+        While passkeys are WebAuthn credentials, WebAuthn credentials are not always passkeys.
+
+    private key
+        A cryptographic key that is kept secret by the owner. It’s used to decrypt data or create a digital signature.
+
+    public key
+        A cryptographic key that can be shared with anyone. It’s used to encrypt data or verify a digital signature.
+
+    relying party
+        The entity that relies on the Web Authentication standard to authenticate users. This refers to you, as you are using Web Authentication to authenticate your users.
+
+    resident key
+        An alternative term for a for a passkey or resident key. It refers to a WebAuthn credential that allows authentication without the server providing credential IDs.
+
+    second-factor authentication
+        The second stage in a two-factor authentication process, where a user provides identification through a biometric sensor, PIN, or pattern.
+
+    two-factor authentication
+        An authentication policy that requires a user to present two proofs of identity. The first is usually a password and the second is frequently tied to some physical device in the user’s possession.
+
+    web authentication
+        This is an open web standard developed by the World Wide Web Consortium (W3C). It describes an authentication standard that enables websites to negotiate the creation of public-key-based credentials with a browser
+
+    WebAuthn
+        A shorthand term for web authentication standard. In this package, some classes include WebAuthn in their names, but the term itself refers to the broader web authentication standard.
+
+    WebAuthn credential
+        A term defined in the Web Authentication specification. It's a key-based credential generated during a WebAuthn registration process, consisting of a public-private :term:`key pair`. The private key is securely stored on the user's device, while the public key is stored on the server for verifying future authentications.
+
+        This should not be confused with the WebAuthnCredential model, which only stores the public key and associated metadata.
```

Comment:
Good opportunity for a link to the `WebAuthnCredential` reference page?

Reply:
Are you suggesting that we create a reference page for `WebAuthnCredential`?
Currently, we have 4 pages in the Reference section:

1. Views
2. Helper
3. Models
4. JSON request and response

Reply:
I meant the section we have on `WebAuthnCredential` on the models reference page: https://github.com/Stormbase/django-otp-webauthn/blob/9546ccefde052b81e4034bbaaae805e516e57014/docs/reference/models.rst?plain=1#L8